ACPs/77-reinventing-subnets/README.md

```diff
@@ -1,175 +1,180 @@
 
-Although the fee paid to the Primary Network to operate a Subnet does not go up with the amount of activity on the Subnet, the fixed, upfront cost of setting up a Subnet Validator on the Primary Network deters new projects that prefer smaller, even variable, costs until demand is observed. _Unlike L2s that pay some increasing fee (usually denominated in units per transaction byte) to an external chain for data availability and security as activity scales, Subnets provide their own security/data availability and the only cost operators must pay from processing more activity is the hardware cost of supporting additional load._
+Although the fee paid to the Primary Network to operate a Subnet does not go up with the amount of activity on the Subnet, the fixed, upfront cost of setting up a Subnet validator on the Primary Network deters new projects that prefer smaller, even variable, costs until demand is observed. _Unlike L2s that pay some increasing fee (usually denominated in units per transaction byte) to an external chain for data availability and security as activity scales, Subnets provide their own security/data availability and the only cost operators must pay from processing more activity is the hardware cost of supporting additional load._
 
 Elastic Subnets, introduced in [Banff](https://medium.com/avalancheavax/banff-elastic-subnets-44042f41e34c), enabled Subnet creators to activate Proof-of-Stake validation and uptime-based rewards using their own token. However, this token is required to be an ANT (created on the X-Chain) and locked on the P-Chain. All staking rewards were distributed on the P-Chain with the reward curve being defined in the `TransformSubnetTx` and, once set, was unable to be modified.
 
 With no Elastic Subnets live on Mainnet, it is clear that Permissionless Subnets as they stand today could be more desirable. There are many successful Permissioned Subnets in production but many Subnet creators have raised the above as points of concern. In summary, the Avalanche community could benefit from a more flexible and affordable mechanism to launch Permissionless Subnets.
 
+### A Note on Nomenclature
+
+Going forward, newly created networks should be referred to as L1s, or Avalanche Layer 1s. L1 validator sets created using the flow introduced in this ACP will **not** be a subset of the Primary Network validator set, making Subnets a technical misnomer. Any network created with the pre-existing flow will still be considered a Subnet, as these types of validators must stake and validate the Avalanche Primary Network.
+
+Though Subnets have always managed their own consensus mechanisms, transaction processing, and security protocols directly within their own networks, the added functionality introduced in this ACP will solidify their status as standalone networks, or layer 1 blockchains.
```

Comment:
I'm finding it a bit difficult to discern the difference between Subnets and L1s with these paragraphs. My understanding is a network will still be considered an L1 and not a Subnet when there is an intersection between the network's validator set and the Primary Network's validator set.

I think the distinction needs to be a bit clearer. I took a stab at defining the difference between Subnets and L1s in a way that's clearer to me but feel free to re-word!

```suggestion
Avalanche Subnets are subnetworks validated by a subset of the Primary Network validator set. The new network creation flow outlined in this ACP does not require any intersection between the new network's validator set and the Primary Network's validator set. Moreover, the new networks have greater functionality and sovereignty than Subnets. To distinguish between these two kinds of networks, the community has been referring to these new networks as _Avalanche Layer 1s_, or L1s for short.

All networks created through the old network creation flow will continue to be referred to as Avalanche Subnets.
```